atlasdb-dbkvs/src/test/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/WhereClausesTest.java

```diff
@@ -139,16 +140,16 @@ public void usesDifferentTableIdentifier() {
 
         List<String> expectedClauses =
                 ImmutableList.of("other.row_name >= ?", "other.row_name < ?", "other.col_name IN (?)");
-        assertEquals(whereClauses.getClauses(), expectedClauses);
+        assertThat(expectedClauses).isEqualTo(whereClauses.getClauses());
 
         checkWhereArguments(whereClauses, ImmutableList.of(START, END, COL1));
     }
 
     private void checkWhereArguments(WhereClauses whereClauses, List<byte[]> expectedArgs) {
         List<Object> actualArgs = whereClauses.getArguments();
         for (int i = 0; i < actualArgs.size(); i++) {
-            assertArrayEquals(expectedArgs.get(i), (byte[]) actualArgs.get(i));
+            assertThat((byte[]) actualArgs.get(i)).isEqualTo(expectedArgs.get(i));
         }
-        assertEquals(expectedArgs.size(), actualArgs.size());
+        assertThat(actualArgs.size()).isEqualTo(expectedArgs.size());
```

Comment:
hasSameSizeAs

Reply:
This should turn into `hasSameSizeAs` if the automation runs a few more times. We opted to implement the simplest migrations such that we could funnel the code to an eventually ideal state rather than cluttering the migrations with attempts to make everything ideal on the first try.

Reply:
Yeah, I left these more as notes to myself to update them :)

Reply:
Though makes sense that running these transformations multiple times would converge on the correct result